src/test/munit/console-redirect/console-redirect.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:apikit="http://www.mulesoft.org/schema/mule/mule-apikit"
+  xmlns:http="http://www.mulesoft.org/schema/mule/http"
+  xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+  xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+  xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://www.mulesoft.org/schema/mule/core"
+  xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+    http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+  	http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+    http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+    http://www.mulesoft.org/schema/mule/mule-apikit http://www.mulesoft.org/schema/mule/mule-apikit/current/mule-apikit.xsd">
+
+  <apikit:config name="console-redirect" raml="munit/console/console-api.raml"
+    outboundHeadersMapName="outboundHeaders" httpStatusVarName="httpStatus"/>
+
+  <munit:config name="console-redirect-test-suite">
+    <munit:parameterizations>
+      <munit:parameterization name="ONPREM">
+        <munit:parameters>
+          <munit:parameter propertyName="fullDomain" value=""/>
+          <munit:parameter propertyName="assertHost" value="localhost"/>
+        </munit:parameters>
+      </munit:parameterization>
+      <munit:parameterization name="CH">
+        <munit:parameters>
+          <munit:parameter propertyName="fullDomain" value="ch-https-41-demo.au-s1.cloudhub.io"/>
```

Comment:
when is this set up as a system property? I see it being used in a logger only.
Shouldn't be a `before-test` with a `Utils::setSystemProperty("fullDomain", "${fullDomain}")`?

Reply:
Nevermind, parameters are set through system properties